language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,1114 @@
+{
```

Comment:
<p>I'd probably shy away from using the term "NLP" tasks as it can get confused with the Natural Language API integration with BQML</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>updated, ta</p>